src/main/java/seedu/taassist/logic/parser/UnassignCommandParser.java

```diff
@@ -22,25 +22,17 @@ public class UnassignCommandParser implements Parser<UnassignCommand> {
      * @throws ParseException if the user input does not conform the expected format
      */
     public UnassignCommand parse(String args) throws ParseException {
-        ArgumentMultimap argMultimap =
-                ArgumentTokenizer.tokenize(args, PREFIX_MODULE_CLASS);
-
+        ArgumentMultimap argMultimap = ArgumentTokenizer.tokenize(args, PREFIX_MODULE_CLASS);
         if (!isPrefixPresent(argMultimap) || argMultimap.getPreamble().isEmpty()) {
-            throw new ParseException(String.format(MESSAGE_INVALID_COMMAND_FORMAT, UnassignCommand.MESSAGE_USAGE));
+            throw new ParseException(String.format(MESSAGE_INVALID_COMMAND_FORMAT, AssignCommand.MESSAGE_USAGE));
```

Comment:
```suggestion
            throw new ParseException(String.format(MESSAGE_INVALID_COMMAND_FORMAT, UnassignCommand.MESSAGE_USAGE));
```
Seems to be a accidental miscopy?

Reply:
Yep, thanks for pointing out!